mobile/v1/snowflake/sql-runner/sql/standard/99-migrations/1.1.0-migration/mobile-events-staged.sql

```diff
@@ -0,0 +1,42 @@
+-- Create Procedure to drop mobile_events_staged if it is empty
+CREATE OR REPLACE PROCEDURE {{.scratch_schema}}.drop_mobile_events_staged_if_empty()
+  RETURNS VARCHAR
+  LANGUAGE JAVASCRIPT
+  AS
+  $$
+    var getRowCount = snowflake.createStatement(
+           {
+           sqlText: "SELECT count(*) FROM {{.scratch_schema}}.mobile_events_staged{{.entropy}};"
+           }
+        );
+	
+    var res = getRowCount.execute();
+    res.next();
+    row_count = res.getColumnValue(1);
+    if (row_count > 0) {
+        throw "Ensure that all data has been processed from events_staged and it is empty.";
+    }
+    else {
```

Comment:
I don't think this is dropping the table here.

Reply:
🤦 Good spot. Fixed.